docs/source/testing-and-debugging.md

```diff
@@ -0,0 +1,99 @@
+# Testing and Debugging
+
+## Configuring the Debugger
+Neo3-boa is compatible with the [Neo Debugger](https://github.com/neo-project/neo-debugger).
+Debugger launch configuration example:
+```
+{
+    //Launch configuration example for Neo3-boa.
+    //Make sure you compile your smart-contract before you try to debug it.
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "example.nef",
+            "type": "neo-contract",
+            "request": "launch",
+            "program": "${workspaceFolder}\\example.nef",
+            "operation": "main",
+            "args": [],
+            "storage": [],
+            "runtime": {
+                "witnesses": {
+                    "check-result": true
+                }
+            }
+        }
+    ]
+}
+```
+
+It's necessary to generate the nef debugger info file to use Neo Debugger.
+
+### Using CLI
+
+```shell
+$ neo3-boa path/to/your/file.py -d|--debug
+```
+
+### Using Python Script
+
+```python
+from boa3.boa3 import Boa3
+
+Boa3.compile_and_save('path/to/your/file.py', debug=True)
+```
+
+## Neo Test Runner
+
+### Downloading
+
+Install [Neo-Express](https://github.com/neo-project/neo-express#neo-express-and-neo-trace) and [Neo Test Runner](https://github.com/ngdenterprise/neo-test#neo-test-runner).
+
+```shell
```

Comment:
We should refer to each tool installation page instead of showing the commands because the installation varies in each operational system